x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts

```diff
@@ -22,6 +22,8 @@ const configSchema = schema.object({
       'Loaded Kibana', // Sent once per page refresh (potentially, once per session)
       'Hosts View Query Submitted', // Worst-case scenario 1 every 2 seconds
       'Host Entry Clicked', // Worst-case scenario once per second - AT RISK,
+      'Host Flyout Filter Removed', // Worst-case scenario once per second - AT RISK,
+      'Host Flyout Filter Added', // Worst-case scenario once per second - AT RISK,
```

Comment:
I'm still concerned that we keep adding more and more custom events to FS.

From the last comment in https://github.com/elastic/obs-infraobs-team/issues/965, I thought @roshan-elastic's comment was to leverage FS's automatic collection (when analyzing this data from FS POV) and only sending custom events to our in-house system.

cc'ing @shahinakmal as he owns the FS platform.

Reply:
@afharo thanks for checking this! I think that was about the other events I suggested using `watched element` for (Break down the host fly-out view by which tab is shown). Those 2 events were the only ones I actually added as custom events because here we need to track the name of the filter as part of the event and have a breakdown based on it ( I couldn't find a way to have the filter name with a selector or any other way)